litmus-portal/frontend/src/views/Home/ResilienceScoreComparisonPlot/index.tsx

```diff
@@ -0,0 +1,357 @@
+        range: [-10, 110],
+      },
+      cliponaxis: true,
+      layer: 'below_traces',
+      autosize: false,
+      margin: {
+        autoexpand: false,
+        l: 30,
+        r: 30,
+        t: 30,
+        b: 105,
+      },
+      font: {
+        family: 'Ubuntu, monospace',
+        color: palette.customColors.black(0.4),
+      },
+    };
+    setPlotLayout(layout);
+  };
+
+  const redirect = () => {
+    tabs.changeWorkflowsTabs(3);
+    history.push('/workflows');
+  };
+
+  useEffect(() => {
+    processData();
+    processLayout();
+  }, [currentGranularity]);
+
+  return (
+    <div style={{ alignContent: 'center' }}>
+      <div className={classes.flexDisplay}>
+        <div className={classes.adjust}>
+          <Score
+            score={edgeData.highScore}
+            high
+            color={edgeData.highColor}
+            name={edgeData.highName}
+          />
+        </div>
+        <Score
+          score={edgeData.lowScore}
+          high={false}
+          color={edgeData.lowColor}
+          name={edgeData.lowName}
+        />
+        <FormControl
+          variant="outlined"
+          className={classes.formControl}
+          color="secondary"
+          focused
+        >
+          <InputLabel
+            htmlFor="outlined-selection-granularity"
+            className={classes.root}
+          >
+            Granularity
+          </InputLabel>
+          <Select
+            value={currentGranularity.name}
+            onChange={handleChangeInGranularity}
+            label="Granularity"
+            inputProps={{
+              name: 'name',
+              id: 'outlined-selection-granularity',
+            }}
+            className={classes.root}
+          >
+            <MenuItem value="Hourly">Hourly</MenuItem>
+            <MenuItem value="Daily">Daily</MenuItem>
+            <MenuItem value="Monthly">Monthly</MenuItem>
+          </Select>
+        </FormControl>
+        <Tooltip title="Analytics">
+          <IconButton
+            aria-label="Analytics"
+            onClick={redirect}
+            className={classes.analyticsBtnPos}
+          >
+            <AssessmentOutlinedIcon
+              color="secondary"
+              className={classes.analyticsButton}
+            />
+          </IconButton>
+        </Tooltip>
+      </div>
+      <div className={classes.plot}>
+        <Plot
+          data={plotData}
+          layout={plotLayout}
+          useResizeHandler
+          style={{
+            width: 'fit-content',
+            margin: 'auto',
+          }}
+          config={{
+            displaylogo: false,
+            autosizable: true,
+            responsive: true,
+            frameMargins: 0.2,
+            showAxisDragHandles: true,
+            showAxisRangeEntryBoxes: true,
+            showTips: true,
+            displayModeBar: false,
+            toImageButtonOptions: {
+              format: 'png',
+              filename: 'Top4_ResilienceScores_Comparison',
```

Comment:
I hope this the file name to export the plot...if that is so...it needs to be unique each time

Reply:
Okay attaching a timestamp to it.